Mathlib/Geometry/Manifold/ContMDiffMFDeriv.lean

```diff
@@ -69,9 +69,15 @@ protected theorem ContMDiffWithinAt.mfderivWithin {x₀ : N} {f : N → M → M'
     (hf : ContMDiffWithinAt (J.prod I) I' n (Function.uncurry f) (t ×ˢ u) (x₀, g x₀))
     (hg : ContMDiffWithinAt J I m g t x₀) (hx₀ : x₀ ∈ t)
     (hu : MapsTo g t u) (hmn : m + 1 ≤ n) (h'u : UniqueMDiffOn I u) :
+    haveI : IsManifold I 1 M := .of_le (le_trans le_add_self hmn)
```

Comment:
These are annoying, but perhaps that's the price of such a major refactoring. It need not block in this, in any case.